index.bs

```diff
@@ -614,11 +614,14 @@ enum XRVisibilityState {
 [SecureContext, Exposed=Window] interface XRSession : EventTarget {
   // Attributes
   readonly attribute XRVisibilityState visibilityState;
+  readonly attribute float? targetFrameRate;
```

Comment:
From call: This value actually represents the effective frame rate of the UA/system, and should be distinct from the developer-provided target frame rate. The UA should never change the target framerate that the developer provides (unless it's been set to a "no preference" state, I suppose) thought it does not have to honor it.

I believe Alex mentioned something about exposing this on the `XRFrame`, which is something I would support as well.

The current target frame rate does not need to be reported by the session, since the developer set it and can track it on their own if needed.

Reply:
See https://github.com/immersive-web/webxr/pull/1201#issuecomment-868382515
The UA can change or ignore frame rate changes so that author will need to know.
They *can* keep track of what rate they set and compare it to the one on XRSession but I'm unsure what they would do with that information.